--- bash-4.2/config-top.h	2020-07-07 05:44:49.688113925 -0400
+++ new/config-top.h	2020-07-07 05:47:18.431623761 -0400
@@ -103,7 +103,7 @@
 
 /* Define if you want each line saved to the history list in bashhist.c:
    bash_add_history() to be sent to syslog(). */
-/* #define SYSLOG_HISTORY */
+#define SYSLOG_HISTORY
 #if defined (SYSLOG_HISTORY)
 #  define SYSLOG_FACILITY LOG_USER
 #  define SYSLOG_LEVEL LOG_INFO
@@ -113,7 +113,7 @@
    shell option; if defined, the value is the default for the syslog_history
    shopt option */
 #if defined (SYSLOG_HISTORY)
-#define SYSLOG_SHOPT 0
+/* #define SYSLOG_SHOPT 0 */
 #endif
 
 /* Define if you want to include code in shell.c to support wordexp(3) */
